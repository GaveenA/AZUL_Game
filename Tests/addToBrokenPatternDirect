#Saving game infomation
SEED=427940514
#Saving tile bag and box lid infomation
BAG=YBBLLRLUYYRBYUYUYYBYLULULYBRRBRBLYBLBRYYRRBYUBRYULYBYUUUUYBYLLBLURLLRYLBRURLRLUR
LID=
#Saving all factories, factory 0 is the center factory
FACTORY_0=Y
FACTORY_1=
FACTORY_2=
FACTORY_3=
FACTORY_4=
FACTORY_5=
#Saving player 1's infomation
PLAYER_1_NAME=Gav
PLAYER_1_SCORE=0
PLAYER_1_PATTERN_LINE0=R
PLAYER_1_PATTERN_LINE1=BB
PLAYER_1_PATTERN_LINE2=BBB
PLAYER_1_PATTERN_LINE3=LLL
PLAYER_1_PATTERN_LINE4=
PLAYER_1_FLOOR_LINE=
PLAYER_1_MOSAIC_0=-----
PLAYER_1_MOSAIC_1=-----
PLAYER_1_MOSAIC_2=-----
PLAYER_1_MOSAIC_3=-----
PLAYER_1_MOSAIC_4=-----
#Saving player 2's infomation
PLAYER_2_NAME=Debi
PLAYER_2_SCORE=0
PLAYER_2_PATTERN_LINE0=R
PLAYER_2_PATTERN_LINE1=UU
PLAYER_2_PATTERN_LINE2=RRR
PLAYER_2_PATTERN_LINE3=UUUU
PLAYER_2_PATTERN_LINE4=
PLAYER_2_FLOOR_LINE=F
PLAYER_2_MOSAIC_0=-----
PLAYER_2_MOSAIC_1=-----
PLAYER_2_MOSAIC_2=-----
PLAYER_2_MOSAIC_3=-----
PLAYER_2_MOSAIC_4=-----
#Saving the current players turn
CURRENT_PLAYER=2