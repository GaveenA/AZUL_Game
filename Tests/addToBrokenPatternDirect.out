#Saving game infomation
SEED=427940514
#Saving tile bag and box lid infomation
BAG=YBBLLRLUYYRBYUYUYYBYLULULYBRRBRBLYBLBRYYRRBYUBRYULYBYUUUUYBY
LID=BBBURRUUUY
#Saving all factories, factory 0 is the center factory
FACTORY_0=F
FACTORY_1=RRLU
FACTORY_2=RRLU
FACTORY_3=RYBL
FACTORY_4=RLLU
FACTORY_5=BLLL
#Saving player 1's infomation
PLAYER_1_NAME=Gav
PLAYER_1_SCORE=3
PLAYER_1_PATTERN_LINE0=
PLAYER_1_PATTERN_LINE1=
PLAYER_1_PATTERN_LINE2=
PLAYER_1_PATTERN_LINE3=LLL
PLAYER_1_PATTERN_LINE4=
PLAYER_1_FLOOR_LINE=
PLAYER_1_MOSAIC_0=--R--
PLAYER_1_MOSAIC_1=-B---
PLAYER_1_MOSAIC_2=--B--
PLAYER_1_MOSAIC_3=-----
PLAYER_1_MOSAIC_4=-----
#Saving player 2's infomation
PLAYER_2_NAME=Debi
PLAYER_2_SCORE=3
PLAYER_2_PATTERN_LINE0=
PLAYER_2_PATTERN_LINE1=
PLAYER_2_PATTERN_LINE2=
PLAYER_2_PATTERN_LINE3=
PLAYER_2_PATTERN_LINE4=
PLAYER_2_FLOOR_LINE=
PLAYER_2_MOSAIC_0=--R--
PLAYER_2_MOSAIC_1=----U
PLAYER_2_MOSAIC_2=----R
PLAYER_2_MOSAIC_3=-U---
PLAYER_2_MOSAIC_4=-----
#Saving the current players turn
CURRENT_PLAYER=2